Src/IronPython/Runtime/Operations/IntOps.cs

```diff
@@ -174,12 +174,12 @@ public static partial class Int32Ops {
         }
 
         [StaticExtensionMethod]
-        public static object __new__(CodeContext/*!*/ context, PythonType cls, IList<byte> s) {
+        public static object __new__(CodeContext/*!*/ context, PythonType cls, IList<byte> s, int @base=10) {
```

Comment:
As discussed on gitter, we can move to optional parameters like this since the framework supports it directly.